public/gameInfo.php

```diff
@@ -1146,16 +1146,16 @@ function ResetProgress() {
                         }
 
                         if ($claimExpiration) {
-                            $claimFormattedDate = $claimExpiration->format('d M Y, H:i');
                             $isAlreadyExpired = Carbon::parse($claimExpiration)->isPast();
+                            $isAlreadyExpiredCopy = $isAlreadyExpired ? "Expired" : "Expires";
 
-                            echo "<p>";
-                            if ($isAlreadyExpired) {
-                                echo "Expired on: $claimFormattedDate (" . $claimExpiration->diffForHumans() . ")";
-                            } else {
-                                echo "Expires on: $claimFormattedDate (" . $claimExpiration->diffForHumans() . ")";
-                            }
-                            echo "</p>";
+                            $claimFormattedDate = $claimExpiration->format('d M Y, H:i');
+                            $claimTimeAgoDate = $permissions >= Permissions::JuniorDeveloper
+                                ? "(" . $claimExpiration->diffForHumans() . ")"
+                                : null;
```

Comment:
Should the false result be empty string instead of null?

Reply:
`null` seems to function correctly. Stylistically in PHP does this make more sense as an empty string?

Reply:
It's being used as a string below, which means it has to be cast to a string, and `(string)null` is empty string, so why not do it yourself?

I know php is pretty lenient, but a ternary operator should return the same type of variable from both paths.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1473/commits/45bb321a9dea191606bde2485ce23fa2380d07ee